mne/minimum_norm/inverse.py

```diff
@@ -1177,6 +1177,18 @@ def apply_inverse_epochs(epochs, inverse_operator, lambda2, method="dSPM",
         Additional options for eLORETA. See Notes of :func:`apply_inverse`.
 
         .. versionadded:: 0.16
+    delayed : bool
+        If False, the source time courses are computed. If True, they are
+        stored as a tuple of two smaller arrays in order to save memory. In
+        this case, the first array in the tuple corresponds to the "kernel"
+        shape (n_vertices, n_sensors) and the second array to the "sens_data"
+        shape (n_sensors, n_times). The full source time courses field will be
+        automatically computed when stc.data is called for the first time
+        (see for example: :class:`mne.SourceEstimate`). `delayed=True` is
+        only implemented for fixed orientations (e.g. if `pick_ori="normal"`),
+        and will be ignored for loose orientations. Defaults to False.
```

Comment:
For now if you don't want to go all the way to enable it for `vector` then it should at least raise an error if you say `delayed=True` in a mode that we do not (yet) support

Reply:
But what about the previous part?
```python
        if is_free_ori:
            # Compute solution and combine current components (non-linear)
            sol = np.dot(K, e[sel])  # apply imaging kernel
            if pick_ori != 'vector':
                logger.info('combining the current components...')
                sol = combine_xyz(sol)
```
The `combine_xyz` after `np.dot` looks tricky to me...